src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -113,20 +113,6 @@ internal virtual BoundExpression BindSwitchExpressionCore(SwitchExpressionSyntax
 
         private BoundPattern BindDiscardPattern(DiscardPatternSyntax node, TypeSymbol inputType, bool hasErrors, DiagnosticBag diagnostics)
```

Comment:
>bool hasErrors, DiagnosticBag diagnostics [](start = 97, length = 41)

The parameters `hasErrors` and `diagnostics` are not used.

Reply:
I could have sworn I deleted these parameters but apparently not.

---
In reply to: [230263871](https://github.com/dotnet/roslyn/pull/30847#discussion_r230263871) [](ancestors = 230263871)